### PR TITLE
Bug 16

### PR DESCRIPTION
**Pull request by sparthaje**
# Properly handle power switching
Issue #16 

## Description
Added a thread to contain the `power_watchdog` method.

## Steps to Reproduce
1. Change `get_battery_bus_volts` in eps/_ _ init _ _.py to 
        `from random import randint`
        `power_levels = [8.2, 7.6, 0]`
        `pl = randint(0, 2)`
        `return power_levels[pl]`

2. Run `start` from core module

## Affected Areas
* `core/__init__.py` (added the thread)

### CODE REVIEW

**Review by abagali1 — APPROVED** [review]
Great Work. Merge your changes.

File: core/__init__.py [hunk 1]
```diff
@@ -171,8 +170,11 @@ def start():
 
     enter_normal_mode()
     logger.debug("Entering main loop")
+
+    # Monitor Power Level
+    power_monitoring_thread = ThreadHandler(target=partial(power_watchdog,
```
Comment (abagali1) [inline core/__init__.py]: you are passing too many arguments to `partial` it should be 

`ThreadHandler(target=partial(power_watchdog), name='monitoring_power', parent_logger=logger)`

**Review by abagali1 — CHANGES_REQUESTED** [review]

File: core/__init__.py (hunk 1 above)
Comment (abagali1) [reply core/__init__.py]: Sorry for prematurely approving your changes, I missed this one change

**Comment by abagali1** [comment]
also, can you delete `.pylintrc` and the `scripts/` directory before merging

File: core/__init__.py (hunk 1 above)
Comment (sparthaje) [reply core/__init__.py]: Thanks, I fixed it

**Review by abagali1 — APPROVED** [review]
Good. Merge and delete your branch

**Pull request merged**